digdag-server/src/test/java/io/digdag/server/rs/ProjectResourceTest.java

```diff
@@ -0,0 +1,65 @@
+package io.digdag.server.rs;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import utils.CommandStatus;
+import utils.TemporaryDigdagServer;
+
+import java.nio.file.Path;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static utils.TestUtils.copyResource;
+import static utils.TestUtils.main;
+
+public class ProjectResourceTest
```

Comment:
Please move this test class to `digdag-tests/src/test/java/` and rename package and class with `acceptance` and `LimitProjectArchiveFileSizeIT`. `IT` means integration tests in Digdag.